site/docs/windows.md

```diff
@@ -165,7 +167,9 @@ To build C++ targets, you need:
     The first command sets the path to Visual Studio (BAZEL\_V<b>S</b>), the other
     sets the path to Visual C++ (BAZEL\_V<b>C</b>).
 
-    For Visual Studio 2017, with a default install, instead you might want
+    For Visual Studio 2017 & 2019, besides setting one of `BAZEL_VC` or `BAZEL_VS`, you can specify
```

Comment:
I think this is important and should be mentioned with the bullet points above.
Here it's just buried with other info.

Reply:
Done